### ISSUE TITLE
about the learning rate

### ISSUE THREAD

**Issue by heurainbow** [xu-ji/IIC#11]
I see in your commands that all experiments are done with lr=1e-4 and much more epochs. I wonder why since _deepcluster_ only train on imagenet for 200 epochs with an initial lr=0.05. Does this make much difference on the model performance?

**Comment by xu-ji** [comment]
Hi, yes, it does make a difference. Deepcluster is of course a different objective and different architecture, so it is unsurprising the learning rate is different. We didn't do a huge exploration of learning rates, to be honest. I did some very coarse testing with 1e-3, 4, 5 in the beginning and just chose one that seemed to work well with our loss. And to keep things simple we avoided learning rate schedules. Learning rate is an important hyperparameter, as always. For example with larger effective batch sizes (more diverse data) it is known that learning rates need to be higher to compensate, perhaps this is one difference with deepcluster. As for the epochs, the command line options are just upper limits for the experiment. STL10 for example converged in 268 epochs.
